### DISCUSSION TITLE
Development setup [Feb 09]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5127] [Discord Threads]
> 🙋‍♂️ **geraldodev**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### try node 16 n make sure sdk part is built successfully  
#  
> 🙋‍♂️ **geraldodev**  
#### i ll try node 16, with node 18 I was sure that sdk part was build without errors  
#  
> 🙋‍♂️ **geraldodev**  
#### node 16 did the trick, thank you <@495466997304262658>  
#  
> 🙋‍♂️ **geraldodev**  
#### i have one hanging around to support copilot  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1062299541522624563)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
